modules/cnf-about-numa-aware-scheduling.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_content-type: CONCEPT
+[id="cnf-about-numa-aware-scheduling_{context}"]
+= About NUMA-aware scheduling
+
+NUMA architecture allows a CPU with multiple memory controllers to use any available memory across CPU complexes regardless of where the memory is located. This allows for increased flexibility at the expense of performance. A CPU processing a workload using memory that is outside its NUMA zone is slower than a workload processed in a single NUMA zone. Also, for I/O-constrained workloads, the network interface on a distant NUMA zone slows down how quickly information can reach the application. High-performance workloads, such as telecommunications workloads, cannot operate to specification under these conditions. To process latency-sensitive or high-performance workloads efficiently, all the requested cluster compute resources (CPUs, memory, devices) should be aligned in the same NUMA zone.
```

Comment:
NUMA architecture allows a CPU with multiple memory controllers to use any available memory across CPU complexes,